internal/server/server.go

```diff
@@ -135,30 +143,30 @@ func (s *Server) AcceptConns() error {
 				s.clientConnWg.Done()
 			}()
 
-			s.logger.Println("Accepting new connection")
+			s.logger.Info("Accepting new connection")
 			// Populating the conn map.
 			s.connMut.Lock()
 			s.connMap[conn] = struct{}{}
 			s.connMut.Unlock()
 
 			if err := s.handleConn(c); err != nil && err != ErrCancelComplete {
-				s.logger.Printf("error while handling conn: %v\n", err)
+				s.logger.Error("Error while handling conn", logr.Err(err))
 			}
 		}(conn)
 	}
 }
 
 func (s *Server) Reload(cfg config.Config) {
-	s.logger.Println("Reloading config.. ")
+	s.logger.Info("Reloading config.. ")
```

Comment:
Any reason why we're using two dots instead of three?
```suggestion
	s.logger.Info("Reloading config...")
```